src/macros.jl

```diff
@@ -2161,3 +2161,277 @@ julia> @by df :a begin
 macro by(x, what, args...)
     esc(by_helper(x, what, args...))
 end
+
+##############################################################################
+##
+## @unique - distinct row selection
+##
+##############################################################################
+
+function unique_helper(x, args...)
+    exprs, outer_flags = create_args_vector(args...)
+    t = (fun_to_vec(ex; no_dest=true, outer_flags=outer_flags) for ex in exprs)
+    quote
+        $DataFrames.unique($x, $(t...))
+    end
+end
+
+"""
+    @unique(x, args)
+
+Select unique rows in `AbstractDataFrame`s.
```

Comment:
Can you also communicate that `args` can be transformations? 

What happens if multiple arguments are passed?  Currently it errors 

```
julia> @unique(df, :x, :b)
ERROR: MethodError: no method matching unique(::DataFrame, ::Symbol, ::Symbol)
```

Maybe just allow one argument for consistency? Or we need to do some other implementation which uses `select` under the hood and allows multiple arguments. 

I think that's the best way forward.